ani-cli

```diff
@@ -209,8 +212,14 @@ download_dir="${ANI_CLI_DOWNLOAD_DIR:-.}"
 quality="${ANI_CLI_QUALITY:-best}"
 case "$(uname)" in
 	Darwin*) player_function="${ANI_CLI_PLAYER:-iina}" ;;
-	*) player_function="${ANI_CLI_PLAYER:-mpv}" ;;
+	*) if uname -a | grep -q 'steamdeck';then
+        player_function="${ANI_CLI_PLAYER:-flatpak_mpv}"
+    else
+        player_function="${ANI_CLI_PLAYER:-mpv}"
+    fi ;;
```

Comment:
since having flatpak mpv support is useful outside of steamdeck users (unlike iina, which is OSX only and unlike am launch which is android only) I'd consider making it an argument rather than dependant on the platform string

Reply:
We can default to mpv.
If it is not found check for flatpak mpv and use it if found.
In my opinion it would be easier for users. It shouldnt be a problem because both pacage managers have the same version and no exclusive features, so it makes no sense to have both installed at the same time.